docs/testing-guidelines/testing-guidelines.md

```diff
@@ -106,15 +106,15 @@ Currently, we have a minuscule number of tests which are run by using xUnit.
 ## Running tests outside of CI
 
 When working on new features or fixes, it is natural to want to run those tests locally before making a PR.
-Three helper functions are part of the build.psm1 module to help with that:
+Two helper functions are part of the build.psm1 module to help with that:
 
-* `Restore-PSPester` will restore Pester, which is needed to run `Start-PSPester`
 * `Start-PSPester` will execute all Pester tests which are run by the CI system
 * `Start-PSxUnit` will execute the available xUnit tests run by the CI system
 
 Our CI system runs these as well; there should be no difference between running these on your dev system, versus in CI.
 
-Make sure that you run `Restore-PSPester` before running `Start-PSPester`, or it will fail to run.
+Make sure that you run `Save-Module -Name Pester -Path $PSHOME\Modules` before running `Start-PSPester`
```

Comment:
This isn't quite accurate.  It should be the $PSHome of the process that is started by Start-PSPester.